src/FFB.py

```diff
@@ -1,27 +1,42 @@
 import discord
 from discord.utils import get
 import os
+import logging
 
 FreeFolk_Voice_ID = int(os.environ['VOICE_ROLE_ID'])
 
+def set_up_logging():
+    logging.basicConfig(level=os.getenv("LOG_LEVEL", "DEBUG").upper())
+    log = logging.getLogger("FolkBot")
+
+    return log
 
 class MyClient(discord.Client):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     async def on_voice_state_update(self, member, before, after):
+        LOGGER.debug(f"following data for event happend: \n "
+                     f"User: {member} \n "
+                     f"State Before: {before} \n "
+                     f"State After:{after}")
```

Comment:
Shouldn't there be an extra space? for formatting?

Reply:
Ran the pep8 test and this wasn't a issue ?

Reply:
it was meant more as a output styling issue ```f"State After: {after}")```